CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug in the **validate** command failed on "edit layout" files.
```

Comment:
what is "edit layout" files?

Reply:
There are several types of layout: _edit_,  _details_ , _close_, and more..

Reply:
And it fails only in edit files